ansible/roles/ocp4_workload_rhacm/tasks/workload.yml

```diff
@@ -0,0 +1,78 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- name: Ensure there is connectivity to OCP cluster as system:admin
+  command: oc whoami
+  register: whoami
+  changed_when: false
+
+- name: Fail if not system:admin
+  fail:
+    msg: "Not connected to OCP as system:admin"
+  when: whoami.stdout != "system:admin"
+
+- name: Ensure required templates are present
+  template:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    mode: 0666
+  loop:
+    - { src: ./templates/subscription.j2, dest: /tmp/subscription.yaml }
+    - { src: ./templates/multiClusterHub.j2, dest: /tmp/multiClusterHub.yaml }
+    - { src: ./templates/operatorGroup.j2, dest: /tmp/operatorGroup.yaml }
+
+- name: Ensure RHACM Namespace is present
+  shell: oc create namespace "{{ ocp4_workload_rhacm_acm_project }}"
+
+- name: Ensure RHACM OperatorGroup is present
+  shell: oc -n "{{ ocp4_workload_rhacm_acm_project }}" apply -f /tmp/operatorGroup.yaml
+
+- name: Ensure RHACM Subscription is present
+  shell: oc -n "{{ ocp4_workload_rhacm_acm_project }}" apply -f /tmp/subscription.yaml
+
+# Wait for the operator deployment
+- name: Wait until RHACM Subscription is Ready
+  command: oc -n "{{ ocp4_workload_rhacm_acm_project }}" get csv "{{ ocp4_workload_rhacm_acm_csv }}" -o jsonpath='{.status.phase}'
```

Comment:
using `k8s_info` you can do the same a bit more natively. See examples in the serverless, logging or elasticsearch workloads (roles_workloads directgory!)

Reply:
done.